# Assignments for "Basic Data Types" 

- Suppose you invested in Bitcoin at the end of 2017 when Bitcoin gained a lot of value. What would be your money at the end of a week if you had invested \$1000 with an average daily increase of 12\% ? You can solve the problem using Python.

In [ ]:
##### Help

# Create a variable capital ($1000)

# Create a variable for daily growth (12%)

# Create a variable for period (7)

# Calculate the final growth rate

# Calculate result

# Print result

- Print the text in quotes with Python. However, you must get the numbers from variables using `.format()` notation. <br> Because the text is long, you might consider writing in two lines:

        `"When we buy bitcoin with 1000 USD at the beginning of the week, we would earn 1210.68 USD at the end of the week, with an average gain of 12\%."`

In [83]:
money=1000
rate=12
period=7
newmoney=money*(1+(rate/100))**period
profit=newmoney-money
print (round(x,2))
print ('`"When we buy bitcoin with {} USD at the beginning of the wee \n k, we would earn {} USD at the end of the week, with a  n average gain of {}/%."\''.format(money,round(profit,2),rate))

2210.68
`"When we buy bitcoin with 1000 USD at the beginning of the wee 
 k, we would earn 1210.68 USD at the end of the week, with a  n average gain of 12/%."'


In [41]:
root=input("Enter the temperature in Fahrenheit :")
celcius=(5/9)*(float(root)-32)
print('user --->',root)
print('output --->Temperature (C) :',round(celcius,2))

Enter the temperature in Fahrenheit :26
user ---> 26
output --->Temperature (C) : -3.33


#### - Get the temperature in Fahrenheit from user and write a code to convert it to Celcius. For conversion, you can use this formula: C = (5/9) * (F - 32)

        Enter the temperature in Fahrenheit: 
        user --> 26
        output --> Temperature (C) : -3.33

In [68]:
root=int(input("Enter a three digit number please :"))
digit1 = (root )//100
digit2 = (root - digit1*100)//10
digit3 = (root - digit1*100 - digit2*10)
sum_0f_digit=digit1+digit2+digit3
#sum(int(digit) for digit in str(root))
print('user --->',root)
print('output ---> \"The sum of digits in the number is {}'.format(sum_0f_digit))

Enter a three digit number please :365
user ---> 365
outout ---> "The sum of digits in the number is 14


- Get a three digit number the from user and calculate the sum of the digits in the integer.

        user --> 365
        output --> "The sum of digits in the number is 14

In [81]:
a=int(input("enter the vertical side lenth :"))
b=int(input("enter the horizontal side lenth :"))
c=int(((a**2)+(b**2))**(1/2))
print("\"The length of the hypotenuse is {}".format(c))

enter the vertical side lenth :6
enter the horizontal side lenth :8
"The length of the hypotenuse is 10


- Write some code to calculate the hypotenuse of a right angled triangle. Get the side lengths from the user.

        user --> first side lenth : 6
        user --> first side lenth : 8
        output --> "The length of the hypotenuse is 10